code/game/objects/random/guns.dm

```diff
@@ -58,3 +58,13 @@
 	name = "low chance random gun part"
 	icon_state = "gun-black-low"
 	spawn_nothing_percentage = 75
+
+/obj/spawner/gun/onestar
+	name = "random  One Star gun"
+	icon_state = "gun-orange"
+	tags_to_spawn = list(SPAWN_GUN_OS)
+
+/obj/spawner/gun/onestar/low_chance
+	name = "low chance random  One Star gun"
```

Comment:
```suggestion
	name = "random One Star gun"
	icon_state = "gun-orange"
	tags_to_spawn = list(SPAWN_GUN_OS)

/obj/spawner/gun/onestar/low_chance
	name = "low chance random One Star gun"
```
Copypasted in too many spaces here, sry.

Reply:
making it two words will make it infinitely more annoying to use in SDMM but sure